go-runtime/compile/schema.go

```diff
@@ -495,16 +593,15 @@ func visitGenDecl(pctx *parseContext, node *ast.GenDecl) {
 		if err != nil {
 			pctx.errors.add(err)
 		}
-		if maybeVisitTypeEnumVariant(pctx, node, directives) {
-			return
-		}
+		maybeVisitTypeEnumVariant(pctx, node, directives)
```

Comment:
Is this intentional?

Reply:
This allowed this case: https://github.com/TBD54566975/ftl/issues/1492
Updating it now to collect errors for subsequent type enums it conforms to

Reply:
Actually no. This is there because even though something is a variant we still need to continue to process it's declaration. eg: a type alias that conforms to am enum.